cmd/agent/subcommands/flare/command.go

```diff
@@ -278,6 +280,17 @@ func makeFlare(flareComp flare.Component,
 		err     error
 	)
 
+	streamLogParams := streamlogs.CliParams{
+		FilePath: commonpath.DefaultStreamlogsLogFile,
+		Duration: cliParams.streamLogsDuration,
+		Quiet:    true,
+	}
+
+	// Check if Duration is set but withStreamLogs is not.
+	if streamLogParams.Duration != 60 && !cliParams.withStreamLogs {
```

Comment:
There is an edge cases here where if the user sets the duration to `60s` this error won't be thrown. 

Instead of setting the default in the CLI args setup, maybe you can check if this setting is explicitly configured by the user? 

Something like:
(Pseudocode)
```

if streamLogParams.Duration != nil && !cliParams.withStreamLogs {
    // error
}

duration = 60 * time.second // default case
if streamLogParams.Duration != nil {
    duration = streamLogParams.Duration 
}

streamLogParams := streamlogs.CliParams{
....
```